helm/cilium/templates/default-policies/job.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if .Values.defaultPolicies.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cilium-create-default-policies
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: cilium-create-default-policies
+  annotations:
+    helm.sh/hook: post-install
+    helm.sh/hook-weight: "1"
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+spec:
+  backoffLimit: 50
+  template:
+    metadata:
+      labels:
+        app: cilium-create-default-policies
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ .Values.serviceAccounts.defaultPolicies.name | quote }}
+      priorityClassName: system-cluster-critical
+      {{- with .Values.defaultPolicies.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 6 }}
+      {{- end }}
+      volumes:
+      - name: policies
+        configMap:
+          name: cilium-default-policies
+      initContainers:
+      - name: wait-crds
+        image: "quay.io/giantswarm/docker-kubectl:latest"
+        imagePullPolicy: IfNotPresent
+        command:
+        - sh
+        args:
+        - -c
+        - |
+          for crd in ciliumnetworkpolicies.cilium.io ciliumclusterwidenetworkpolicies.cilium.io
+          do
+            while ! kubectl get crd $crd
```

Comment:
I'd rather rely on `kubectl api-resources`. If it's there you know the CRD is already registered.

Reply:
Sorry I missed that one. Not really `k api-resources --api-group="cilium.io"`